internal/metrics/metrics.go

```diff
@@ -239,6 +242,10 @@ func NewMetrics(registry *prometheus.Registry) *Metrics {
 			},
 			[]string{"kind", "error"},
 		),
+		CircuitBreakerSettings: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: ContourCircuitBreakerSettings,
+			Help: "Number of times a worker process has been restarted",
```

Comment:
Help text should be updated here to be relevant

Reply:
yeah yeah you right, fixed now!